crates/bevy_ui/src/render/mod.rs

```diff
@@ -187,6 +201,7 @@ pub fn extract_atlas_uinodes(
         >,
     >,
 ) {
+    let out = extracted_uinodes.next_buffer();
```

Comment:
Nit: I like short names, but since it's only used once, could we get a more descriptive name here?

```suggestion
    let nodes_buffer = extracted_uinodes.next_buffer();
```